docs/reference/streamfield/blocks.md

```diff
@@ -0,0 +1,462 @@
+(streamfield_block_reference) =
+
+# StreamField block reference
+---
```

Comment:
does not need a `---` after the `#` as this will introduce a new line or not be parsed correctly.
```suggestion
```